package.json

```diff
@@ -1,38 +1,44 @@
 {
-  "name": "eslint-plugin-netguru-ember",
-  "version": "1.6.5",
-  "description": "Eslint plugin for Netguru Ember Style Guide",
+  "name": "eslint-plugin-ember",
+  "version": "1.0.0",
```

Comment:
Will have to be `2.0.0` for my eye, as `1.0.0` up to `1.1.0` were already released on NPM: https://www.npmjs.com/package/eslint-plugin-ember .